### ISSUE TITLE
CookieBot Policy being filtered for Websites

### ISSUE THREAD

**Issue by virdual** [uBlockOrigin/uAssets#11661]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://virdual.com/cookie-policy/`

### Describe the issue

Simply put, the Cookie Policy page hosted by Cookie Bot is falsely filtered from the site (and questionably other sites that use Cookie Bot). The odd part is, the page is embedded and in no way a popup. Testing was completed on an unmodified browser profile using default settings for uBlock. 

### uBlock Origin version

uBlock Origin 1.40.8

### Browser name and version

Chrome Version 98.0.4758.82

### Settings

n/a

**Comment by mapx-** [comment]
It's filtered by `AdGuard Annoyances` and by `Fanboy’s Annoyance` (or `EasyList Cookie`) because is considered an annoyance.

**Comment by Procyon-b** [comment]
> an unmodified browser profile using default settings

With default settings I can see the dialog. (tested on 2 computers)

**Comment by krystian3w** [comment]
This not an ad so not blocked in stock enabled lists.

No need uBo syntax so filter made by AdGuard or EasyList shouldn't  copied to optional uBo Annoyances.

Also now harder found clear cookie bar without injected GDPR in same node.
